core/src/main/java/jeeves/config/springutil/JeevesNodeAwareLogoutSuccessHandler.java

```diff
@@ -84,8 +87,12 @@ protected String determineTargetUrl(HttpServletRequest request, HttpServletRespo
 
                     String siteHost = settingManager.getValue(Settings.SYSTEM_SERVER_HOST);
                     String siteProtocol = settingManager.getValue(Settings.SYSTEM_SERVER_PROTOCOL);
-                    int sitePort = settingManager.getValueAsInt(Settings.SYSTEM_SERVER_PORT);
-
+                    
+                    // some conditional logic to handle the case where there's no port in the settings
+                    int sitePort = Geonet.DefaultHttpPort.HTTP;
+                    if (StringUtils.isNumeric(Settings.SYSTEM_SERVER_PORT)) {
```

Comment:
Reviewing this code is checking a wrong value, the name of the setting. I think should be like:

```suggestion
                    if (StringUtils.isNumeric(settingManager.getValue(Settings.SYSTEM_SERVER_PORT))) {
```